scripts/hail_batch/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -0,0 +1,62 @@
+"""Launch analysis runner for all cell types and chromosomes"""
+
+import re
+import os
+from analysis_runner.cli_analysisrunner import run_analysis_runner
+from google.cloud import storage
+
+REFERENCE = 'GRCh38'
+BUCKET_NAME = 'cpg-tob-wgs-test'
+INPUT_PATH = f'gs://{BUCKET_NAME}/full_data/{REFERENCE.lower()}/analysis_output/kccg'
+
+
+# get cell types
+cs_client = storage.Client()
```

Comment:
I think you should wrap the "doing bits" of this script in a function, and call it from the __main__ python thing, just a better habit rather than doing it in the root of the script, eg:

```python
def submit_eqtl_jobs():
    # do stuff here

if __name__ == '__main__':
    submit_eqtl_jobs()
```

Reply:
I've now added this in 🤞Out of curiosity, how does calling from `main.py` work when you have two launch scripts (`launch_generate_eqtl_spearman.py` and `launch_round2.conditional_analysis.py`)?